packages/cf-style-container/src/index.js

```diff
@@ -42,6 +42,7 @@ const applyTheme = (ComponentToWrap, ...themes) => {
     }
   }
 
+  ThemedComponent.displayName = `Themed${ComponentToWrap}`;
```

Comment:
Is `ComponentToWrap` a string? How does React coerce a component class to a string?